From 6a3c8709435f35899e9ab98a2422b6f66fe1b413 Mon Sep 17 00:00:00 2001
From: Russ Allbery <rra@stanford.edu>
Date: Tue, 20 Dec 2011 17:13:29 -0800
Subject: Fix the ASCII fallback string for AE

* cpan/podlators/lib/Pod/Man.pm: Fix the ASCII fallback string for the AE
ligature to use the string that was actually defined.

Bug-Debian: http://bugs.debian.org/652851
Origin: http://git.eyrie.org/?p=perl/podlators.git;a=commit;h=0d83c47faa30ebc8980eebbf43d26463b8a56d7c

Patch-Name: fixes/podlators_ae_ligature_fallback.diff
---
 cpan/podlators/lib/Pod/Man.pm |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cpan/podlators/lib/Pod/Man.pm b/cpan/podlators/lib/Pod/Man.pm
index 96f3fcc..376c611 100644
--- a/cpan/podlators/lib/Pod/Man.pm
+++ b/cpan/podlators/lib/Pod/Man.pm
@@ -1321,7 +1321,7 @@ sub parse_from_filehandle {
     undef, undef, undef, undef,            undef, undef, undef, undef,
     undef, undef, undef, undef,            undef, undef, undef, undef,
 
-    "A\\*`",  "A\\*'", "A\\*^", "A\\*~",   "A\\*:", "A\\*o", "\\*(AE", "C\\*,",
+    "A\\*`",  "A\\*'", "A\\*^", "A\\*~",   "A\\*:", "A\\*o", "\\*(Ae", "C\\*,",
     "E\\*`",  "E\\*'", "E\\*^", "E\\*:",   "I\\*`", "I\\*'", "I\\*^",  "I\\*:",
 
     "\\*(D-", "N\\*~", "O\\*`", "O\\*'",   "O\\*^", "O\\*~", "O\\*:",  undef,
